rskj-core/src/test/java/co/rsk/peg/feeperkb/FeePerKbSupportImplTest.java

```diff
@@ -171,21 +213,25 @@ void save() {
     }
 
     private Transaction getTransactionFromUnauthorizedCaller(SignatureCache signatureCache) {
-        ECKey unauthorizedKey = ECKey.fromPublicOnly(Hex.decode("0305a99716bcdbb4c0686906e77daf8f7e59e769d1f358a88a23e3552376f14ed2"));
-        RskAddress unauthorizedCallerAddress = new RskAddress(unauthorizedKey.getAddress());
-
         Transaction txFromUnauthorizedCaller = mock(Transaction.class);
-        when(txFromUnauthorizedCaller.getSender(signatureCache)).thenReturn(unauthorizedCallerAddress);
+        when(txFromUnauthorizedCaller.getSender(signatureCache)).thenReturn(this.getUnauthorizedRskAddress());
 
         return txFromUnauthorizedCaller;
     }
 
     private Transaction getTransactionFromAuthorizedCaller(SignatureCache signatureCache) {
-        RskAddress authorizedCallerAddress = new RskAddress("a02db0ed94a5894bc6f9079bb9a2d93ada1917f3");
-
         Transaction txFromAuthorizedCaller = mock(Transaction.class);
-        when(txFromAuthorizedCaller.getSender(signatureCache)).thenReturn(authorizedCallerAddress);
+        when(txFromAuthorizedCaller.getSender(signatureCache)).thenReturn(this.getAuthorizedRskAddress());
 
         return txFromAuthorizedCaller;
     }
+
+    private RskAddress getUnauthorizedRskAddress(){
+        ECKey unauthorizedKey = ECKey.fromPublicOnly(Hex.decode("0305a99716bcdbb4c0686906e77daf8f7e59e769d1f358a88a23e3552376f14ed2"));
+        return new RskAddress(unauthorizedKey.getAddress());
```

Comment:
```suggestion
        return new RskAddress("e2a5070b4e2cb77fe22dff05d9dcdc4d3eaa6ead");
```
No need for the key, we can use the address directly

Reply:
Done!